ape_trezor/client.py

```diff
@@ -0,0 +1,108 @@
+from typing import Optional, Tuple
+
+from ape.api.accounts import TransactionAPI
+from ape.types import TransactionSignature
+from eth_typing.evm import ChecksumAddress
+from trezorlib import ethereum
+from trezorlib.client import get_default_client  # type: ignore
+from trezorlib.exceptions import PinException
+from trezorlib.tools import parse_path as parse_hdpath  # type: ignore
+from trezorlib.transport import TransportException
+
+from ape_trezor.exceptions import TrezorAccountException, TrezorUsbError
+from ape_trezor.hdpath import HDAccountPath, HDBasePath
+
+
+class TrezorClient:
+    """
+    This class is a client for the Trezor device.
+    """
+
+    def __init__(self, hd_root_path: HDBasePath):
+        try:
+            self._client = get_default_client()
+        except TransportException as exc:
+            message = (
+                "Unable to open Trezor device path."
+                "Make sure you have your device unlocked via the passcode "
+                "and have the Ethereum app open."
+            )
+            raise TrezorUsbError(message) from exc
+        self._hd_root_path = hd_root_path
+
+    def get_account_path(self, account_id: int) -> str:
+        try:
+            return ethereum.get_address(
+                self._client, parse_hdpath(f"{self._hd_root_path}/{account_id}")
+            )
+        except PinException as exc:
```

Comment:
We should also catch `trezorlib.exceptions.TrezorFailure` and wrap it in our own.

I got this error: `trezorlib.exceptions.TrezorFailure: DataError: Forbidden key path`

I just set up my Trezor and am messing up on accident lol but it is a good test.

Reply:
This happened because I used a number in my passphrase and could not figure out how to input a number on the Trezor text device